src/proto/TheOpenNetwork.proto

```diff
@@ -44,7 +44,7 @@ message Transfer {
     uint32 mode = 5;
 
     // Expiration UNIX timestamp (optional, now() + 60 by default)
-    uint64 expire_at = 6;
+    uint32 expire_at = 6;
```

Comment:
TON requires `uint32`, to prevent unexpected overflow, we decided to make it `uint32` in proto as well.